src/dismisal_preference.js

```diff
@@ -0,0 +1,6 @@
+chrome.runtime.onMessage.addListener((request,sender,sendResponse)=>{
+    if (!!request.message && request.message == 'dont_show_again_preference'){
+        //insert preference value into storage
+        chrome.storage.local.set({dont_show_again_preference: request.value})
+    }
```

Comment:
What does this `!!` operator do?

Reply:
!! is equivalent to if request.message is not undefined or if request.message has a valid value